versioned_docs/version-2.0.x/reference/targets/relocated_files.mdx

```diff
@@ -17,14 +17,14 @@ For example, you can relocate `src/resources/project1/data.json` to instead be
 `dependencies` field, rather than using the original `files` target.
 
 To remove a prefix:
-
 ```
 # Results in `data.json`.
 relocated_files(
     file_targets=["src/resources/project1:target"],
     src="src/resources/project1",
     dest="",
 )
+
 ```
```

Comment:
It looks like the spacing has gone a bit weird here.

Reply:
Fixed in https://github.com/pantsbuild/pantsbuild.org/pull/53

Reply:
Although, how did you re-generate the docs here? That red line should've been fixed by `prettier`. Did you run` prettier` after running the generation command?

Reply:
I didn't; I didn't realise I had to :)

Reply:
If there was a way to have it run in the script definition (e.g. run the generation and format) id +1 that

Reply:
Done in https://github.com/pantsbuild/pantsbuild.org/pull/65#pullrequestreview-1805284823